pkg/test/runner/config.go

```diff
@@ -76,6 +76,11 @@ type TestCaseConfig struct {
 	//    after test.
 	Debug bool `json:"debug,omitempty" yaml:"debug,omitempty"`
 
+	// Scripts is a BASH script which will be run instead of the default
+	// kpt commands. All the metrics applied to kpt commands are also applied to
+	// this script. The PWD for this script will be set to the copied package.
+	Script string `json:"script,omitempty" yaml:"script,omitempty"`
+
```

Comment:
Another way to implement this would be, if `exec.sh` or `test.sh` script exists in the test config directory, then copy that script in the `pkgPath` and execute it. One benefit of that will be avoiding the content from a script to this YAML config during the test development.

I was reviewing a test PR and we need a way to run script for setting up the test (for ex. ensuring this image doesn't exist in the local docker registry). I think eventually mechanism can be extending by supporting `setup.sh` and `teardown.sh` script along with `exec.sh`.

WDYT ?

Reply:
That sounds good. Will do it in that way.

Reply:
done